README.md

```diff
@@ -501,6 +611,40 @@ ovs-vsctl add-port ovs_pvp_br0 vhost0 -- \
           set Interface vhost0 ofport_request=2
 ```
 
+__For Dual NUMA node system__
+For OVS-DPDK we would like to use the second Hyper Thread pair (CPU 2,30) for
+the PMD threads. And the third Hyper Thread pair (CPU 4,32) for the none PMD
+DPDK threads. To configure this we execute the following commands:
+
+```
+ovs-vsctl set Open_vSwitch . other_config:dpdk-init=true
+ovs-vsctl set Open_vSwitch . other_config:dpdk-socket-mem=2048
+ovs-vsctl set Open_vSwitch . other_config:pmd-cpu-mask=0x00008002
```

Comment:
Don't think the masks are correct here.

MASK: 0x40000004
MASK: 0x100000010

Also a note here how to setup the memory if the card was on numa node 1 would help.